.github/release-drafter.yml

```diff
@@ -0,0 +1,51 @@
+name-template: "$RESOLVED_VERSION"
+tag-template: "$RESOLVED_VERSION"
+
+categories:
+  - title: "Backwards Incompatible Changes"
+    labels:
+      - "backwards incompatible"
+
+  - title: "Features"
+    labels:
+      - "feature"
+      - "enhancement"
+      # usually combined with enhancement (avoid double listing)
+      - "deprecation"
+
+  - title: "Bug Fixes"
+    labels:
+      - "bug"
+      - "bug fix"
+
+  - title: "Other Changes"
+    labels:
+      - "dependency"
+      - "docs"
+      - "trivial"
+      - "maintenance"
+
+change-template: "- $TITLE (#$NUMBER) @$AUTHOR"
```

Comment:
If the change text comes from a PR title, then we should update the [pull request template](https://github.com/jazzband/pip-tools/blob/master/.github/PULL_REQUEST_TEMPLATE.md) accordingly.